modifications/DataClass.cc

```diff
@@ -61,21 +72,21 @@ void DataClass::readyReadToken()
 {
     QNetworkReply* reply = qobject_cast<QNetworkReply*>(sender());
     QString replyStr = reply->readAll();
-    QJsonDocument jsonResponse = QJsonDocument::fromJson(replyStr.toUtf8());
-    QJsonObject jsonObject = jsonResponse.object();
-    accessToken = jsonObject["access_token"].toString();
-    if(accessToken!=""){
-        emit tokenGenerated();
-    }
-    else{
+    int statusCode = reply->attribute( QNetworkRequest::HttpStatusCodeAttribute).toInt();
+    if(statusCode == 201 || statusCode == 200){
+        QJsonDocument jsonResponse = QJsonDocument::fromJson(replyStr.toUtf8());
+        QJsonObject jsonObject = jsonResponse.object();
+        accessToken = jsonObject["access_token"].toString();
+        if(accessToken!=""){
+            emit tokenGenerated();
+        }
+    }else{
         emit tokenNotGenerated();
+        showMessage("Unable to Connect to Server", statusCode);
     }
 }
-
+//*******************************************************************************************************************************************************************************
 //DRONE STATUS CHECK
-/// GET REQUEST
-/// For more information refer "registry/aircraft/rfm/" end point
-/// For more information visit: https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/openskies-sh/aerobridge/master/api/aerobridge-1.0.0.resolved.yaml
```

Comment:
Why not keep the API Docs link here on top where you were saying API calls begin? Would be nice to have this reference. Maybe even check against an aerobridge API version for compatibility since the current implementation is done for API version 1.0.0. @hrishiballal thoughts?